client/signup/steps/plans/index.jsx

```diff
@@ -215,6 +246,22 @@ PlansStep.propTypes = {
 	translate: PropTypes.func.isRequired,
 	planTypes: PropTypes.array,
 	flowName: PropTypes.string,
+	isMonthlyPricingTest: PropTypes.bool,
+};
+
+const PlansStepWithMonthlyPricingTest = ( props ) => {
+	const isEligibleForMonthlyPricing = 'onboarding' === props.flowName;
+
+	return (
+		<Experiment name="monthly_pricing_test_phase_1">
+			<DefaultVariation name="control">
+				<PlansStep { ...props } isMonthlyPricingTest={ false } />
+			</DefaultVariation>
+			<Variation name="treatment">
+				<PlansStep { ...props } isMonthlyPricingTest={ isEligibleForMonthlyPricing } />
+			</Variation>
+		</Experiment>
```

Comment:
This could ruin the experiment.

The analysis is going to report the users who `isEligibleForMonthlyPricing == false` as being in the experiment and experiencing the treatment, but really they're experiencing the control. So they'll bias the experiment.

Instead, I propose the following:
```suggestion
		(!isEligibleForMonthlyPricing 
			? <PlanStep { ...props } isMonthlyPricingTest={ false } />
			: <Experiment name="monthly_pricing_test_phase_1">
			<DefaultVariation name="control">
				<PlansStep { ...props } isMonthlyPricingTest={ false } />
			</DefaultVariation>
			<Variation name="treatment">
				<PlansStep { ...props } isMonthlyPricingTest={ true } />
			</Variation>
		</Experiment>)
```

Reply:
Actually, I just realized we also put them in the experiment on the server-side, so you'll need to remove the server-side variation retrieval and leave eligibility specifically for the client, or remove eligibility being handled in the client.

Reply:
@withinboredom We no longer use the `<Experiment>` component. Instead, we query the experiments with `<QueryExperiments />` and pull the data using hooks.